test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -7366,7 +7366,7 @@ BOOST_AUTO_TEST_CASE(array_length_cannot_be_constant_function_parameter)
 			}
 		}
 	)";
-	CHECK_ERROR(text, TypeError, "Constant identifier declaration must have a constant value.");
```

Comment:
This error message is also not too helpful, because you cannot assign a constant value to this variable.